core/src/types/read/reader.rs

```diff
@@ -133,7 +133,7 @@ impl Reader {
                     let size = self
                         .ctx
                         .accessor()
-                        .stat(self.ctx.path(), OpStat::new())
+                        .stat(self.ctx.path(), get_stat_op_from_read_op(self.ctx.args()))
```

Comment:
At current stage, only version is meaningful. How about just extract version? We can remove `get_stat_op_from_read_op`.

Reply:
ok, but my opinion is because the reader will call this to get content length, i think both of the `OpStat` and `OpRead` should be the same.

Reply:
> i think both of the `OpStat` and `OpRead` should be the same.

The reader might send chunked read request, so they might be different.

Reply:
> > i think both of the `OpStat` and `OpRead` should be the same.
> 
> The reader might send chunked read request, so they might be different.

ok